resources-generator/src/main/kotlin/dev/icerock/gradle/generator/apple/AppleImagesGenerator.kt

```diff
@@ -60,16 +61,23 @@ class AppleImagesGenerator(
             val imagesContent = validFiles.joinToString(separator = ",\n") { file ->
                 val scale = file.scale
 
+                // language=js
```

Comment:
why here js language mark? i not see js here

Reply:
removed